platform/ios/app/MBXViewController.m

```diff
@@ -251,39 +251,6 @@ - (void)viewDidLoad
     self.mapView.experimental_enableFrameRateMeasurement = YES;
     self.hudLabel.titleLabel.font = [UIFont monospacedDigitSystemFontOfSize:10 weight:UIFontWeightRegular];
 
-    if ([MGLAccountManager accessToken].length)
-    {
-        self.styleIndex = -1;
-        [self cycleStyles:self];
```

Comment:
Until another PR lands that removes this functionality, we’ll want to keep these two lines.